### DISCUSSION TITLE
"Workspace" should be an option under "Move Tab", it should not be a new top-level item

### DISCUSSION THREAD

**Discussion by jean** [zen-browser/desktop#1322] [Ideas]
As the title says: there should be only one option to "Move Tab", and it should include "Workspace" as an option in addition to Start/End/New Window.
"Change Tab(s) to Workspace" should not be a new top-level item.

**Comment by JSGRD22** [comment]
I disagree as it would require extra mouse movements/waiting to open another submenu. However, it could instead have a divider below the Start/End/New Window options, followed by a list of workspaces.
